daemon/start.go

```diff
@@ -253,7 +252,7 @@ func (daemon *Daemon) Cleanup(container *container.Container) {
 		logrus.Warnf("%s cleanup: failed to unmount secrets: %s", container.ID, err)
 	}
 
-	if err := mount.RecursiveUnmount(container.Root); err != nil {
```

Comment:
Hmm, now when I look at this line... isn't it supposed to be below `container.UnmountVolumes`?

cc @cpuguy83

Reply:
This code was added by commit eaa5192856c, at that time `container.UnmountVolumes` was already there (below)

Reply:
@cpuguy83 could you have a look?

Reply:
`container.Root` is `/var/lib/docker/containers/<id>`

Reply:
What about volumes? I haven't checked but I suppose volumes are mounted under the `container.Root`, so it makes sense to have volumes unmounted first. Or am I missing something?

Surely it's a separate issue; just wondering.